shell/src/main/java/alluxio/cli/docgen/ConfigurationDocGenerator.java

```diff
@@ -47,9 +46,36 @@
   private static final String CSV_FILE_DIR = "docs/_data/table/";
   private static final String YML_FILE_DIR = "docs/_data/table/en/";
   public static final String CSV_FILE_HEADER = "propertyName,defaultValue";
+  public static final String TEMP_PREFIX = "temp-";
```

Comment:
private instead of public?
same for the identical variable in the MetricsDocGenerator